stage2/03-screenly/01-run.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+# Add checkout to production branch later
+
+on_chroot << EOF
+  git clone https://github.com/Screenly/screenly-ose.git /home/pi/screenly
```

Comment:
This would install the development branch (master) instead of the production version.